resolwe_bio/processes/amplicon/bwa_trim.yml

```diff
@@ -9,12 +9,12 @@
     expression-engine: jinja
     executor:
       docker:
-        image: resolwebio/dnaseq:4.0.0
+        image: resolwebio/dnaseq:4.1.0
     resources:
       memory: 16384
       cores: 10
   data_name: "Alignment ({{ reads|sample_name|default('?') }})"
-  version: 1.2.0
+  version: 1.2.1
```

Comment:
You have bumped middle version number in all processes except for this one. This is just an observation, not a request for change. Feel free to fix or leave as it is.

Reply:
Yes, all other processes jumped from resolwebio/dnaseq:3.x.x. `bwa_trim` already was on v4.0.0